tailwind.config.mjs

```diff
@@ -27,6 +30,7 @@ export default {
       },
       fontSize: {
         base: ['13px', '1.4'],
+        lg: ['14px'],
```

Comment:
Do you have a rough idea of what the different font sizes will look like? Making the "large" font size only one pixel larger than the base seems like a small increment.

Reply:
Right now I'm trying to gather all of the different sizes into one place. This scale is going to be a mess initially because our typography is a bit of a mess. You're right, these are silly sizes. But I don't want to make any actual visual changes yet until we've corralled the situation.